py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -508,3 +511,119 @@ def pdist(
         )
     indices = np.triu_indices(shape[0], k=1)
     return impl.select.index(ctx, target, source_ir, f"{name}_index", norm, indices)
+
+
+def cdist_forward(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    x1: TRTTensor,
+    x2: TRTTensor,
+    p: float,
+    compute_mode: int,
```

Comment:
Switch to `Optional[int]`, since `None` is a valid choice

Reply:
Thank you for your comment! I've made the requested changes.